add Stackify's Retrace Template

mattjbrooksii opened the pull request:
committing the Stackify's Retrace API Template to the Octopus Community Templates. Retrace gives developer teams visibility to critical application performance and behavior data so they can deploy better applications faster. This template allows the user to send a request easily to an API that will let Retrace know of your deployment.


---

### Step template checklist

- [ ] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [ ] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [ ] Parameter names should not start with `$`
- [ ] **To minimize the risk of step template parameters clashing with other variables in a project that uses the step template, ensure that you prefix your parameter names (e.g. an abbreviated name for the step template or the category of the step template**
- [ ] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

Fixes # . _If there is an open issue that this PR fixes add it here, otherwise just remove this line_

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/OctopusDeploy/Library?pullRequest=637) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/OctopusDeploy/Library?pullRequest=637) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/OctopusDeploy/Library?pullRequest=637) it.</sub>

alfhenrik commented on step-templates/stackify-api-template:
```diff
@@ -0,0 +1,121 @@
+      },
+      "Links": {}
+    }
+  ],
```
Missing `LastModifiedBy` property with :octocat: handle

alfhenrik commented on step-templates/stackify-api-template:
```diff
@@ -0,0 +1,121 @@
+    "ExportedAt": "2018-01-29T22:12:56.674Z",
+    "OctopusVersion": "4.1.8",
+    "Type": "ActionTemplate"
+  }
```
Missing `Category` property with the value `stackify`, would be good to also add a logo for `Stackify` as per our **contribution guide**

```
Optional: Assign your template to an existing category. Have a look at existing templates to find the category that matches your template. If you don't specify it your template will be assigned to 'other' category.
   - If you add a new category, make sure that you add an icon in `.png` format with a size of 200x200px to the `logos` folder with the same name as your category. Also, the `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it.
```

mattjbrooksii commented:
I have made the following changes based on your comments.

- Added LastModifiedBy property
- Added Category "stackfiy"
- Uploaded logo for stackify category

Thank you for your time!

alfhenrik commented:
![](https://media.giphy.com/media/l4JzeAX7an8Zg8q88/giphy.gif)
